opm/simulators/flow/GenericOutputBlackoilModule.cpp

```diff
@@ -492,12 +492,16 @@ assignToSolution(data::Solution& sol)
         DataEntry{"RV",       UnitSystem::measure::oil_gas_ratio,                         rv_},
         DataEntry{"RVSAT",    UnitSystem::measure::oil_gas_ratio,                         oilVaporizationFactor_},
         DataEntry{"SALT",     UnitSystem::measure::salinity,                              cSalt_},
-        DataEntry{"SOMAX",    UnitSystem::measure::identity,                              soMax_},
         DataEntry{"SSOLVENT", UnitSystem::measure::identity,                              sSol_},
-        DataEntry{"SWMAX",    UnitSystem::measure::identity,                              swMax_},
         DataEntry{"WATKR",    UnitSystem::measure::identity,                              relativePermeability_[waterPhaseIdx]},
         DataEntry{"WAT_DEN",  UnitSystem::measure::density,                               density_[waterPhaseIdx]},
         DataEntry{"WAT_VISC", UnitSystem::measure::viscosity,                             viscosity_[waterPhaseIdx]},
+        DataEntry{"SGMAX",    UnitSystem::measure::identity,                              sgmax_},
+        DataEntry{"SOMAX",    UnitSystem::measure::identity,                              soMax_},
+        DataEntry{"SHMAX",    UnitSystem::measure::identity,                              shmax_},
+        DataEntry{"SWMAX",    UnitSystem::measure::identity,                              swMax_},                            
+        DataEntry{"SOMIN",    UnitSystem::measure::identity,                              somin_},
+        DataEntry{"SWHY1",    UnitSystem::measure::identity,                              swmin_},
```

Comment:
Just for simplicity and to assist future maintenance, I think we should keep this list in alphabetically sorted order (on vector name).